pegasus/sites.v3/hourofcode.com/public/how-to/virtual-draft.haml

```diff
@@ -8,13 +8,13 @@ theme: how_to_guide
 = view :"how-to/how_to_banner", nav_audience: "guide_nav_virtual"
 
 -# Intro section
-= view :"how-to/how_to_intro", intro_header: "TBD", intro_desc: "TBD", intro_desc_02: "TBD"
+= view :"how-to/how_to_intro", intro_header: hoc_s(:hoc_how_to_virtual_event_intro_heading), intro_desc: hoc_s(:hoc_how_to_events_intro_desc_01), intro_desc_02: hoc_s(:hoc_how_to_virtual_event_intro_desc_02)
 
 -# Steps
 = view :"how-to/steps/steps_virtual"
 
 -# FAQ
-=view :"how-to/faq/faq_virtual"
+=view :"how-to/faq/faq_default"
```

Comment:
Is `faq_default` already a file or was `faq_virtual` supposed to be renamed to it?

Reply:
`faq_default` already exists — this guide ended up using that one instead of having custom faqs so I deleted the `faq_virtual` file.